apps/src/applab/designElements/button.jsx

```diff
@@ -202,6 +197,14 @@ class ButtonEvents extends React.Component {
   }
 }
 
+// Initial button size when fontSize is 14 or smaller is 80x30 (classic theme)
+// Initial button size when fontSize is 15 or greater is 100x40 (new themes)
+const MAX_SMALL_FONT_SIZE = 14;
+const DEFAULT_BUTTON_WIDTH = '100px';
+const DEFAULT_BUTTON_WIDTH_SMALL = '80px';
+const DEFAULT_BUTTON_HEIGHT = '40px';
+const DEFAULT_BUTTON_HEIGHT_SMALL = '30px';
+
```

Comment:
(section below) All of these themeValues shouldn't be in the element.jsx files. It'd be very difficult to add, remmove, or edit a theme. Instead move these to an ApplabThemeStyles constants file and organize by theme. Then you can do theme.value here. That will also make testing easier as we can write a test that will check that all themes have the relevant values.

Reply:
(See comment on conversation page)